docs/blog/2019-04-03-how-to-build-a-blog-with-wordpress-and-gatsby-part-1/index.md

```diff
@@ -0,0 +1,117 @@
+
+Now let's get down to business...
+
+<video autoplay muted loop width="400">
+  <source src="https://media.giphy.com/media/xUOwGmsFStnxzIGC2s/giphy.mp4">
+</video>
+
+## Setting up WordPress
+
+This article assumes you have at least a rudimentary knowledge of WordPress. If you've never used it, you can check out [my quick guide to setting up a WordPress site](https://www.iamtimsmith.com/blog/how-to-build-a-beautiful-website-in-less-than-10-minutes/).
+
+## A bit about WordPress as a CMS
+
+Wordpress is a Content Management System (or CMS) built using PHP and MySQL. Typically when building a WordPress site, you would build a theme that essentially acts as a "skin" for the content. Since you're using Gatsby to build the frontend, you won't be using this functionality.
+
+As I mentioned in my other blog post, WordPress exposes REST APIs which can be used to get data for your React application. This same principle will be used here, although you'll be adding some things to improve the REST API and make it more flexible.
+
+I also like to use WordPress at a subdomain such as `admin.example.com` so it is totally separate from the actual site.
+
+![Underscores Theme](./images/underscores.png)
+
+## The WordPress theme
+
+Since you are not worried about how the site looks in WordPress, the theme you use doesn't necessarily matter. The only two files you really need in the theme are the `style.css` file to tell WordPress the information about your theme and the `functions.php` file so you can create menus, create custom content types, create default Advanced Custom Fields, etc. Later on, you can add more files to the theme for page templates, but that's totally optional.
+
+When building WordPress sites I generally stick with the [Underscores](https://underscores.me/) theme, so I will use that as the starter today since it already has a lot of stuff set up out-of-the-box.
+
+## The WordPress Plugins
+
+I know I was on my soap box earlier about using a bunch of plugins, and generally I'm not a huge fan of them. There are a few plugins that can really make your life easier as a developer though.
+
+### Advanced Custom Fields
+
+The first plugin I'd like to mention is [Advanced Custom Fields](https://www.advancedcustomfields.com/), or ACF. This is perhaps one of my favorite plugins of all time since it allows you to create custom fields for any post type or page. You can override defaults. You can create any number of fields for the user ranging from What You See Is What You Get (or WYSIWYG) editors and date pickers (free) to galleries and repeater fields (pro). With this tool, you can truly customize the site to be exactly what your client needs.
```

Comment:
"Advanced Custom Fields" is ACF (for short) / is referred to as ACF. Right now, it might confuse people - as it might be understood as those are 2 different plugins (IMO)

Reply:
Added explanation that ACF is the same as Advanced custom fields.